LifeCycleAssessment_Engine/Compute/EvaluateTM65.cs

```diff
@@ -0,0 +1,155 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+using BH.oM.LifeCycleAssessment;
+using BH.oM.LifeCycleAssessment.MaterialFragments;
+using BH.oM.Physical.Materials;
+using System.Collections.Generic;
+using BH.Engine.Matter;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("Returns reinforcement volume.")]
+        [Input("name", "Name of the equipment type being evaluated. This name will be used to identify the complexity multiplier.")]
+        [Input("weight", "Weight of the equipment in kg.")]
+        [Input("quantity", "The amount of determined QuantityType.")]
+        [Input("materialComposition", "The material composition of the object being evaluated. This is a breakdown of materials by ratio.")]
+        [Output("epd", "An Environmental Product Declaration.")]
+        public static EnvironmentalProductDeclaration EvaluateTM65 (string name, double weight, double quantity, QuantityType quantityType, MaterialComposition materialComposition) //add optional refrigerant input after prototype is working.
+        {
+            // TM65 calculation 
+            // TotalWeight x ExplicitBulkByMass x 1.1 (ReplacementWeightIncrease) x Scale Up Factor x 1.3 (Buffer Factor) + RefrigerantTotalFromImageBelow
+
```

Comment:
Add "Other" material option for anything under 100% in ratios for materials, that has specific value in TM65